### DISCUSSION TITLE
Move tabs between workspaces using tree-style tabs

### DISCUSSION THREAD

**Discussion by tollsimy** [Floorp-Projects/Floorp#1280] [Feature Request]
No option available to move one or multiple tabs from a workspace to another when using tree-style tabs vertical bar, while perfectly possible using the normal Floorp bar.
The only way I found to solve the issue is to make Floorp bar visible, move tabs and then hide again Floorp bar.

Tree-style tabs:
![image](https://github.com/user-attachments/assets/59da18a2-d95c-4968-9712-a0ab432f0782)

Default Floorp bar:
![image](https://github.com/user-attachments/assets/9c0da5bb-c227-491a-9598-dbe5bb9fa8a8)

**Comment by lisasnt** [comment]
Yeah, it would be really nice actually

**Comment by JSGRD22** [comment]
I would like this feature too. Right now, the most convenient solution I have to offer (without having to open the vertical tabs sidebar) is to do it from the "list all tabs" interface, but it can only move one tab at a time.
